src/ad-services/live-connect/index.ts

```diff
@@ -48,31 +49,35 @@ export class LiveConnect extends BaseServiceSetup {
 				.then(() => {
 					utils.logger(logGroup, 'loaded');
 					this.resolveAndTrackIds();
+					const customQf = this.instantConfig.get<number>('icLiveConnectQf')?.toString();
+					if (customQf) {
+						this.resolveAndTrackIds(customQf);
+					}
 				});
 		} else {
 			communicationService.emit(eventsRepository.LIVE_CONNECT_CACHED);
 			utils.logger(logGroup, `already loaded and available in ${this.storageConfig.type}Storage`);
 		}
 	}
 
-	resolveAndTrackIds(): void {
+	resolveAndTrackIds(qf?: string): void {
```

Comment:
```suggestion
	resolveAndTrackIds(qf = '0.3'): void {
```
isn't it better and simpler?